### DISCUSSION TITLE
Conda install unavailable?

### DISCUSSION THREAD

**Discussion by PhilipLab** [rsagroup/rsatoolbox#385] [Q&A]
I'm having trouble at the very first step - installing the toolbox into my conda environment so I can run it from a jupyter notebook. Documents say to "conda install -c conda-forge rsatoolbox", but I get the following error:

> PackagesNotFoundError: The following packages are not available from current channels:
> 
>   - rsatoolbox
> 
> Current channels:
> 
>   - https://repo.anaconda.com/pkgs/main/osx-arm64
>   - https://repo.anaconda.com/pkgs/main/noarch
>   - https://repo.anaconda.com/pkgs/r/osx-arm64
>   - https://repo.anaconda.com/pkgs/r/noarch

What am I doing wrong?

**Comment by JasperVanDenBosch** [comment]
hi @PhilipLab 

You can find some information on our Anaconda builds here:

- [package info on anaconda.org](https://anaconda.org/conda-forge/rsatoolbox)
- [our conda-forge repo](https://github.com/conda-forge/rsatoolbox-feedstock)

Indeed it seems we do not currently produce arm64 packages there. I'm not sure why. I work on arm64 myself but typically install from source via `pip install rsatoolbox`. This should also work in your conda environment. Note that the latest version for which we have conda builds is *0.1.4*.

I'm gonna look into this further and reach out to the Continuum Analytics team to for information.
